xml/aliengo.xml

```diff
@@ -61,17 +61,17 @@
             <site name="imu" size="0.0005 0.0005 0.0005" pos="0 0 0" />
 	    <!-- front right -->
 	    <body name="FR_hip" pos="0.2399 -0.051 0">
-		<inertial pos="-0.022191 -0.015144 -1.5e-05" quat="0.0135493 0.706835 -0.0117635 0.707151" mass="1.993" diaginertia="0.00558695 0.00491009 0.00290132" />
+		<inertial pos="-0.022191 -0.015144 -1.5e-05" quat="0.0135493 0.706835 -0.0117635 0.707151" mass="1.993" diaginertia="0.002903894 0.004907517 0.005586944" />
```

Comment:
Could you please describe briefly how you computed the new `diaginertia` and `quat` values?